templates/vue/src/components/tyde/activities/CircleOfSupport/CommunityView/onboarding/ObFinishView.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <cos-modal show>
+    <b-container class="modal-cont">
+      <b-row align-h="center">
+        <b-col
+          v-for="connection in threeConnections()"
+          :key="connection.id"
+          align-self="center"
+          class="ob-connection-box"
+        >
+          <p class="ob-connection">{{ connection.name }}</p>
+          <h1>{{ connection.avatar }}</h1>
+        </b-col>
+      </b-row>
+      <b-row align-h="center">
+        <h3>
+          Great job! Here are some of
+          <br />
+          your connections.
```

Comment:
`<br>` is generally not a very responsive way of coding since smaller screen sizes will have their own line breaks in addition as well. I recommend adding a `max-width` to the element here to achieve the results you want instead.